<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortForm</class>
 <widget class="QWidget" name="SortForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Sort">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>781</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
   </property>
   <property name="title">
    <string>分拣模块</string>
   </property>
   <widget class="QPushButton" name="pushButton_Sort_AdjustIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>矫正进卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort_AdjustOut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>矫正出卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort_OCRReaderIn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OCR识别进卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort_VerifyIn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>核验进卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort_LongOut">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽出卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SortMod_Light">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>闪灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SortMod_Light_Close">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SortModInit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>70</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>卡槽1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>卡槽2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>卡槽3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>卡槽4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_Send">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>440</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receive</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>440</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
